tools/flang2/flang2exe/ll_write.cpp

```diff
@@ -834,18 +814,6 @@ ll_write_function(FILE *out, LL_Function *function, LL_Module *module, bool no_r
   fputs("}\n\n", out);
 }
 
-void
```

Comment:
I wonder if it shouldn't belong to some other PR...

Reply:
This change actually belongs to #1321。

Reply:
> I wonder if it shouldn't belong to some other PR...

This change actually belongs to #1321。